### ISSUE TITLE
Renew Airflow's SSL certificate via the EFF certbot

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#9577]
Airflow's SSL certificate expired last week. This machine is not exposed to the wider internet, so the risk is minimal, but it should be addressed as soon as reasonably acceptable. 

Chia developed programmatic acquisition of a new certificate at last renewal in [this issue](https://github.com/cityofaustin/atd-data-tech/issues/8555). As a result of her work, `certbot` and the route-53 authentication modules are installed on the airflow host, and it should hopefully be a matter of re-executing the command to generate a new certificate and private key in the correct locations and restarting the airflow docker container that uses them. 

The command or short script that is used to do the above would be a welcome addition to our gitbook documentation and a prime candidate to be orchestrated via a Prefect flow.

**Comment by amenity** [comment]
@frankhereford - let's discuss your recommended approach here, assuming we want to take care of it quickly.

**Comment by amenity** [comment]
Will leave the Prefect-ization to another day. Command used will go in Gitbook.

**Comment by frankhereford** [comment]
@amenity wrote:
> Will leave the Prefect-ization to another day. Command used will go in Gitbook.

I may have misremembered that I was to write a placeholder issue for the down-the-road automation of renewal. I wrote a short ticket for it in [9620](https://github.com/cityofaustin/atd-data-tech/issues/9620), but please feel free to close it if it's not helpful. Thank you!!

**Comment by amenity** [comment]
Thanks, @frankhereford !

**Comment by frankhereford** [comment]
The Airflow SSL certificate has been renewed and will remain valid until October 4th, 2022. While previously, we have used snapd to install a snap that provided the certbot utility, I have switched over to using the docker images certbot pushes up to docker hub. 

I have documented the steps taken in a script found in `/root/renew_airflow_ssl_cert.sh` which should be a one-stop-shop that can be executed when it's time to renew. I have not actually tested the script, due to the nature of certbot not being willing to give you a new cert over and over, but I'll be happy to do that test in October.
